src/global/app-css.scss

```diff
@@ -54,7 +54,6 @@ main {
 @import "mixins.scss";
 @import "markdown.scss";
 @import "text.scss";
-@import "headers.scss";
```

Comment:
This file was just styles for `ListingDetailHeader` + one class for `ContentAccordion` and one class that wasn't used, so this file mostly became `ListingDetailHeader.scss`